src/analyses/variable-sensitivity/abstract_environment.cpp

```diff
@@ -824,3 +854,18 @@ exprt assume_greater_than(
 
   return assume_less_than(env, symmetric_expr, ns);
 }
+
+class simplify_vsd_exprt : public simplify_exprt
```

Comment:
I would rather not do this, if possible.

Reply:
We do have to do it, but I've made further changes to try and make things less intrusive.